### ISSUE TITLE
data augmentation

### ISSUE THREAD

**Issue by lolongcovas** [zylo117/Yet-Another-EfficientDet-Pytorch#9]
I saw that the data augmentation only does left|right flip, I wonder if only one single data augmentation achieves the SoTA or the weights/benchmark metrics reported are achieved by this flip as data augmentation. 

Best and thanks for sharing your FAQ, especially the part of your repo vs others.

**Comment by zylo117** [comment]
It does. Horizontal flip is the most effective augmentation for a long time. Even the official efficientdet and their paper used hflip and no other augmentation.

Actually I used lots of augmentations before. But when I got desperate, I disabled them. Guess what, the training loss and the validation loss went down really quick. I think augmentations are sometimes and too complex and beyond the model's comprehension

**Comment by lolongcovas** [comment]
thanks for sharing your experience! good to know... :+1:
